modules/gitops-argo-cd-properties.adoc

```diff
@@ -11,19 +11,28 @@ The Argo CD Custom Resource consists of the following properties:
 [options="header"]
 |===
 |Name |Description |Default |Properties
+
 |`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
 
-|`applicationSet`
-|`applicationSet` controller configuration options.
-| `_<object>_`
-a|* `<image>` - The container image for the `applicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
+|`aggregatedClusterRoles`    |The aggregation of several cluster roles for a cluster-scoped instance. | `__<empty>__` |
+
+|`applicationSet` |The ApplicationSet Controller configuration options. | `_<object>_`
+a|* `<image>` - The container image for the ApplicationSet Controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
   * `<version>` - The tag to use with the `applicationSet` container image.
+  * `<enabled>` - The flag to use to enable the ApplicationSet Controller during Argo CD installation.
+  * `<env>` - Specify the environment for ApplicationSet Controller pods.
   * `<resources>` - The container compute resources.
   * `<logLevel>` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
-  * `<logFormat>` - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
-  * `<parallelismLimit>` - The kubectl parallelism limit to set for the controller (`--kubectl-parallelism-limit` flag).
+  * `<logFormat>` - The log format used by the Argo CD Application Controller component. Valid options are `text` and `json`.
+  * `<parallelismLimit>` - The kubectl parallelism limit to set for the controller (the `--kubectl-parallelism-limit` flag).
   * `sourceNamespaces` - The list of non-control plane namespaces for creating and managing Argo CD `ApplicationSet` resources in target namespaces.
-  * `scmProviders` - The list of URLs of the allowed Source Code Manager (SCM) Providers.
+  * `scmProviders` - The list of URLs of the allowed Source Code Manager (SCM) providers.
+  * `<scmRootCAConfigMap>` - The name of the config map that stores the Gitlab SCM Provider's TLS certificate which will be mounted on the Application Set Controller at the "/app/tls/scm/cert" path.
```

Comment:
```suggestion
  * `<scmRootCAConfigMap>` - The name of the config map that stores the Gitlab SCM Provider's TLS certificate that will be mounted on the Application Set Controller at the "/app/tls/scm/cert" path.
```
From the "Conjunctions" topic in IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=grammar-conjunctions
>Use _that_, without a comma, to introduce a restrictive clause. Use _which_, preceded by a comma, to introduce a nonrestrictive clause.
>
>Consider the following examples:
>
>The system units, which have two drives, are floor-standing models. (Nonrestrictive)
>The system units that have two drives are floor-standing models. (Restrictive)

Reply:
Fixed.